clustercontroller-apps/src/main/java/com/yahoo/vespa/clustercontroller/apps/clustercontroller/ClusterController.java

```diff
@@ -104,4 +95,18 @@ void shutdownController(FleetController controller) throws Exception {
         controller.shutdown();
     }
 
+    /**
+     * Creates a path in zookeeper for this fleetcontroller. Seems like this is meant as a check
+     * that zookeeper server is up and that we are able to do operations against it.
```

Comment:
Consider implementing this by calling CuratorFramework.blockUntilConnected().

Reply:
Much nicer, thanks, fixed.